### PR TITLE
kiosk_mode: android:attr/lStar not found when assembling in release mode #681

### PR DESCRIPTION
**Pull request by ijascookee**
Updated android versions

### CODE REVIEW

**Comment by chaeMil** [comment]
cc @ookami-kb @Sevastyan @alboiuvlad29

File: kiosk_mode/pubspec.yaml
Comment (alboiuvlad29) [inline kiosk_mode/pubspec.yaml:1]: This file should not be changed. It's automatically updated by Melos when we publish the package.

**Review by alboiuvlad29 — CHANGES_REQUESTED** [review]
Thank you very much for the contribution! 🙏

There's one change from my side, otherwise looks good to me.

File: kiosk_mode/pubspec.yaml
Comment (manav-zf) [reply kiosk_mode/pubspec.yaml:1]: Please add this fix

**Comment by manav-zf** [comment]
Please merge the fix @ijascookee  @chaeMil @alboiuvlad29

**Comment by alboiuvlad29** [comment]
Closing in favour of https://github.com/MewsSystems/mews-flutter/pull/695.

**Pull request closed without merging**